### ISSUE TITLE
Invalid argument

### ISSUE THREAD

**Issue by ak1976** [wakatime/vscode-wakatime#73]
It seens some problem after update to 1.2.4

```
/D:/Microsoft VS Code/resources/app/out/vs/workbench/workbench.main.js:294 [Extension Host] [WakaTime] [ERROR] Error: Command failed: pythonw "D:\Microsoft VS Code\data\extensions\wakatime.vscode-wakatime-1.2.4\out\wakatime-master\wakatime\cli.py" --file "e:\My Documents\Visual Studio 2017\Projects\YangMingDevFramework-branch-portal-v2\YM.ILIS.Portal\angular\src\app\_header\header\header.component.ts" --plugin "vscode/1.29.1 vscode-wakatime/1.2.4" --alternate-project YM.ILIS.Portal --config C:\Users\tonysu\.wakatime.cfg --logfile C:\Users\tonysu\.wakatime.log
pythonw: can't open file '"D:\Microsoft VS Code\data\extensions\wakatime.vscode-wakatime-1.2.4\out\wakatime-master\wakatime\cli.py"': [Errno 22] Invalid argument
```

**Comment by alanhamlett** [comment]
This looks like wakatime-cli wasn't able to download successfully. Does a file exist at:
`D:\Microsoft VS Code\data\extensions\wakatime.vscode-wakatime-1.2.4\out\wakatime-master\wakatime\cli.py`

Can you try restarting VS Code in debug mode and watch your Dev Console window to see if it downloads wakatime-cli correctly? If it doesn't work after restarting VS Code, also try re-installing the plugin.

**Comment by DJN1** [comment]
I am having the same issue. It successfully downloads and extracts wakatime-core, but then it throws the same error as above.
<img width="356" alt="wakatime_issue" src="https://user-images.githubusercontent.com/34351129/49827690-87f4a100-fd4f-11e8-99b5-e409103fa00d.PNG">

What can I do to fix this?

**Comment by DJN1** [comment]
What do you mean by proxy config?

**Comment by brunodotcom** [comment]
Hey everyone, it's not a fix but solved for me changing my extensions folder to root, like "C:\vscode\extensions"

It can be done by setting the extension folder path on shortcut properties

![image](https://user-images.githubusercontent.com/20939785/49901198-ad64d600-fe47-11e8-9c46-6415cedc9f67.png)

**Comment by ak1976** [comment]
Just for your information...

I just updated VS Code todat to 1.30, and used it's new function "Install Another Version" in the Extention section, revert WakaTime back to 1.2.3 and everything is working fine now.

**Comment by alanhamlett** [comment]
This should be fixed with 0efd068fd24c2c5638a96e766c3b91711f4e8afb and released today in v1.2.5.
